benchmarking/commons/hpo_main_simulator.py

```diff
@@ -340,7 +344,7 @@ def start_benchmark_simulated_backend(
             )
         tuner_name = experiment_tag
         if configuration.use_long_tuner_name_prefix:
-            tuner_name += f"-{sanitize_sagemaker_name(configuration.benchmark)}-{seed}"
+            tuner_name += f"-{sanitize_sagemaker_name(benchmark_name)}-{seed}"
```

Comment:
Is this line the fix? Are there any other places where we might also have done this?